src/main/java/net/mcreator/ui/blockly/BlocklyEditorType.java

```diff
@@ -19,7 +19,15 @@
 
 package net.mcreator.ui.blockly;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public record BlocklyEditorType(String registryName, String extension, String startBlockName) {
+	private static final Map<String, BlocklyEditorType> TYPES = new HashMap<>();

```

Comment:
I know this is not enforced by the formatting rules (yet), but I prefer, and most of the software has, a new line after class declaration and before class closure at the end of the file.

Reply:
✔️ Fixed